enterprise/api/v1/urls.py

```diff
@@ -197,6 +198,13 @@
         enterprise_group.EnterpriseGroupViewSet.as_view({'post': 'remove_learners'}),
         name='enterprise-group-remove-learners'
     ),
+    re_path(
```

Comment:
(nit): let's rename this to something like `r'^enterprise-customer/(?P<enterprise_uuid>[A-Za-z0-9-]+)/users/?$'`

Reply:
`enterprise-customer` is an already existing endpoint, so it might be difficult to bundle this functionality under that endpoint. Does the rename to `enterprise-customer-user` work instead?
Reference: https://github.com/openedx/edx-enterprise/blob/master/enterprise/api/v1/urls.py#L41

Reply:
Discussed offline - we will go with `enterprise-customer-support`